openassessment/xblock/static/sass/xb/utilities/_reset.scss

```diff
@@ -288,6 +288,14 @@
   }
 
   /**
+   * Set opacity on buttons to 50% (which matches LMS styling).
```

Comment:
@andy-armstrong the pattern library doesn't address buttons that are styled in this way. It only covers buttons where the disabled colors are explicitly set (which is what I patterned _btn--primary_ and _btn--secondary_ off of).

http://ux.edx.org/components/buttons/

These buttons are styled to look like links, with no outlines and backgrounds. Let's chat about how the variety of different buttons will be handled when LMS is using the pattern library.